From a8b378e6cc055a93ec2650a25734d0fd1d4a1629 Mon Sep 17 00:00:00 2001
From: Helge Deller <deller@gmx.de>
Date: Thu, 27 Oct 2022 09:12:05 +0200
Subject: [PATCH 0707/1398] [Backport] parisc: Export iosapic_serial_irq()
 symbol for serial port driver

stable inclusion
from stable-v5.10.154
commit 98f836e80d210312ec98ffbbce4a0b6c4b6dc34e
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I64YCB

Reference: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=98f836e80d210312ec98ffbbce4a0b6c4b6dc34e

--------------------------------

commit a0c9f1f2e53b8eb2ae43987a30e547ba56b4fa18 upstream.

The parisc serial port driver needs this symbol when it's compiled
as module.

Signed-off-by: Helge Deller <deller@gmx.de>
Reported-by: kernel test robot <lkp@intel.com>
Cc: <stable@vger.kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Chen Jiahao <chenjiahao16@huawei.com>
---
 drivers/parisc/iosapic.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/parisc/iosapic.c b/drivers/parisc/iosapic.c
index 8a3b0c3a1e92..fd99735dca3e 100644
--- a/drivers/parisc/iosapic.c
+++ b/drivers/parisc/iosapic.c
@@ -875,6 +875,7 @@ int iosapic_serial_irq(struct parisc_device *dev)
 
 	return vi->txn_irq;
 }
+EXPORT_SYMBOL(iosapic_serial_irq);
 #endif
 
 
-- 
2.17.1

